site/dat/repo/various.json

```diff
@@ -512,5 +512,22 @@
         "downloadUrl":"https://github.com/delirius325/jmeter-elasticsearch-backend-listener/releases/download/2.6.1/jmeter.backendlistener.elasticsearch-2.6.1.jar"
       }
     }
+  },
+  {
+    "id": "camel-straw",
+    "name": "Camel Straw Launcher",
+    "description": "Allows you to run your jmeter load-test on camelstraw.io",
+    "screenshotUrl": "https://gallery.azure.com/artifact/20151001/entechsolutions.camel-straw.1.0.0/Artifacts/SampleImage/849648d3-edd8-47b7-9438-8ec0237637de.png",
+    "helpUrl": "https://github.com/eabdullin/camel-straw-plugin/wiki",
```

Comment:
The wiki page contains no information. Also, the image you add is not screenshot, it's rather promotional image.

The repo of plugins is not the advertisement place, its purpose is to help finding plugins. I'm ok with having plugin that implements integration with camelstraw.io in the repo. But I don't agree to accept something that has no technical documentation and no screenshots.

Reply:
Hello, Andrey! Thanks for reply. Ok, I will add more details and will apply another screenshot.